pom.xml

```diff
@@ -61,7 +66,29 @@
        </exclusions>
    </dependency>
 
-  
+    <dependency>
+      <groupId>javax.activation</groupId>
+      <artifactId>javax.activation-api</artifactId>
+      <version>${version.javax.activation}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jboss.spec.javax.annotation</groupId>
+      <artifactId>jboss-annotations-api_1.3_spec</artifactId>
+      <version>${version.org.jboss.spec.javax.annotation.jboss-annotations-api_1.3_spec}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jboss.spec.javax.xml.bind</groupId>
+      <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+      <version>${version.org.jboss.spec.javax.xml.bind-api_2.3_spec}</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.xml.ws</groupId>
+      <artifactId>jaxws-api</artifactId>
+      <version>${version.jaxws-api}</version>
+    </dependency>
```

Comment:
I guess this is not needed, as there's already a dependency to the JAX-WS API 2.3 above.

Reply:
This archive is required.  There are references to package javax.jws  that this archive provides.

Reply:
the javax.jws package comes from JSR181 API, javax.jws:jsr181-api is a transitive dependency of javax.xml.ws:jaxws-api and is what is providing the stuff you need; I'd say check which GAV we currently use in JBossWS for JSR181 API and include that instead. javax.xml.ws:jaxws-api shouldn't be included given we have our fork that you're correctly already pulling.